packages/ffe-chips-react/src/Chip.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType, ForwardedRef } from 'react';
+import { fixedForwardRef } from './fixedForwardRef';
+import { ChipBase, type ChipBaseProps } from './ChipBase';
+import { Icon } from '@sb1/ffe-icons-react';
+
+const closeOpen400Sm =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjAiIHZpZXdCb3g9IjAgLTk2MCA5NjAgOTYwIiB3aWR0aD0iMjAiPjxwYXRoIGQ9Im0zODktMzY5IDI5OS0yOTlxMTAuOTA5LTExIDI1LjQ1NS0xMVE3MjgtNjc5IDczOS02NjhxMTEgMTEgMTEgMjUuNTgyIDAgMTQuNTgzLTEwLjYwNyAyNS4xODhMNDE1LTI5MnEtMTAuOTA5IDExLTI1LjQ1NSAxMVEzNzUtMjgxIDM2NC0yOTJMMjIxLTQzNXEtMTEtMTEtMTEtMjUuNXQxMS0yNS41cTExLTExIDI1LjY2Ny0xMSAxNC42NjYgMCAyNS4zMzMgMTFsMTE3IDExN1oiLz48L3N2Zz4=';
+
+const checkOpen400Sm =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjAiIHZpZXdCb3g9IjAgLTk2MCA5NjAgOTYwIiB3aWR0aD0iMjAiPjxwYXRoIGQ9Im0zODktMzY5IDI5OS0yOTlxMTAuOTA5LTExIDI1LjQ1NS0xMVE3MjgtNjc5IDczOS02NjhxMTEgMTEgMTEgMjUuNTgyIDAgMTQuNTgzLTEwLjYwNyAyNS4xODhMNDE1LTI5MnEtMTAuOTA5IDExLTI1LjQ1NSAxMVEzNzUtMjgxIDM2NC0yOTJMMjIxLTQzNXEtMTEtMTEtMTEtMjUuNXQxMS0yNS41cTExLTExIDI1LjY2Ny0xMSAxNC42NjYgMCAyNS4zMzMgMTFsMTE3IDExN1oiLz48L3N2Zz4=';
+
+export type ChipProps<As extends ElementType = 'button'> = ChipBaseProps<As> & {
+    /** Kind of chip */
+    kind?: 'default' | 'selectable' | 'removable';
```

Comment:
Dette er subjektivt pirk, men kan `kind` heller være `type` eller `chipType` eller noe sånt? Ordet 'kind' kjennes litt wonky for meg i denne sammenhengen, selv om jeg skjønner hva du mener. 😅

Reply:
`type` er tatt av button feks og det likte ikke typescript. Man må kunne sende `type="button"`. Tenkte også chipType men også tenkte jag att vi vet ju allrede att det er ett chip.... :)

Reply:
Jag kan gjerne byte alltså hvis dere vill men dette var bakgrunden etil att jag falt på kind :)

Reply:
Tror jeg foretrekker chipType over kind! Evt chipVariant? chipStyle?

Reply:
Jag liger chiptype mest av disse selv iaf

Reply:
Jag gikk før chipType